docs/Configuring.md

```diff
@@ -430,28 +349,81 @@ database {
     numThreads = 1
    }
 }
+
+database.metadata {
+  profile = "slick.jdbc.HsqldbProfile$"
+  db {
+    driver = "org.hsqldb.jdbcDriver"
+    url = """
+    jdbc:hsqldb:file:cromwell-executions/cromwell-db/metadata-db;
+    shutdown=false;
+    hsqldb.default_table_type=cached;hsqldb.tx=mvcc;
+    hsqldb.result_max_memory_rows=10000;
+    hsqldb.large_data=true;
+    hsqldb.applog=1;
+    hsqldb.lob_compressed=true;
+    hsqldb.script_format=3
+    """
+    connectionTimeout = 120000
+    numThreads = 1
+   }
+}
 ```
 
-Explanation of the options (see also http://hsqldb.org/doc/2.0/guide/dbproperties-chapt.html):
+To set this up with SQLite the following configuration can be used:
+```hocon
+database {
+  profile = "slick.jdbc.SQLiteProfile$"
+  db {
+    driver = "org.sqlite.JDBC"
+    url = "jdbc:sqlite:file:cromwell-executions/cromwell-db/cromwell-db?foreign_keys=true&date_class=text"
```

Comment:
I just did some testing and found that when the directory does not exist, cromwell will hang. 
A working URL would be
`url = "jdbc:sqlite:file:cromwell-db.sqlite?foreign_keys=true&date_class=text"`